### PR TITLE
Optimize startup

### PR DESCRIPTION
**Pull request by luis-pereira**
It shaves around 8 ms.

### CODE REVIEW

File: daemon/core.cpp [hunk 1]
```diff
@@ -69,6 +69,18 @@ enum
 
 static Core *s_Core = 0;
 
+static const QLatin1String ExecKey("Exec");
+static const QLatin1String EnabledKey("Enabled");
+static const QLatin1String CommentKey("Comment");
+static const QLatin1String serviceKey("service");
+static const QLatin1String pathKey("pathkey");
```
Comment (philippwiesemann) [inline daemon/core.cpp]: The string should be "path" (not "pathkey").

Comment (luis-pereira) [inline daemon/core.cpp]: @philippwiesemann Thanks. Fixed now.

**Comment by luis-pereira** [comment]
Test Test.

**Comment by agaida** [comment]
:)

**Pull request merged**